app/models/manageiq/providers/redhat/infra_manager/provision_workflow.rb

```diff
@@ -108,4 +108,19 @@ def filter_allowed_hosts(all_hosts)
     ovirt_services = ManageIQ::Providers::Redhat::InfraManager::OvirtServices::Builder.new(ems).build(:use_highest_supported_version => true).new(:ems => ems)
     ovirt_services.filter_allowed_hosts(self, all_hosts)
   end
+
+  def validate_memory_limit(_field, values, dlg, fld, _value)
+    limited = get_value(values[:memory_limit])
+    return nil unless limited
+
+    ems_version = source_ems.api_version[/\d+\.\d+/x]
```

Comment:
I think this and the next line should be in a method, like "supported_in_version?" or something like that.